src/store/actions/error.js

```diff
@@ -0,0 +1,8 @@
+import { ERROR } from './types';
```

Comment:
Eslint reported that `ERROR` is not defined in types. Check your `./types` file to make sure it is included.

Reply:
This is no longer needed. I forgot to run npm run eslint. I should just set up some pre-commit hooks just like on the demo blog and the lifion repos. Will open a task/ticket for this.